provider/workspace.go

```diff
@@ -35,6 +35,9 @@ func workspaceDataSource() *schema.Resource {
 			_ = rd.Set("owner", owner)
 
 			ownerEmail := os.Getenv("CODER_WORKSPACE_OWNER_EMAIL")
+			if ownerEmail == "" {
+				ownerEmail = "default"
```

Comment:
would it be safer to set it to `default@domain.tld` or something similar?

Reply:
Yes I think so. We can update.

Reply:
`example.com` is also a good placeholder for a domain name.